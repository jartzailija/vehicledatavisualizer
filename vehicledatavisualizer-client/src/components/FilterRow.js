import React, { Component } from 'react';
import { Row, Col } from 'reactstrap';
import styled from 'styled-components';
import Suggester from '../containers/Suggester';

const FilterRow = () => {

  const StyledCol = styled(Col)`
    padding-top: 20px;
    padding-bottom: 20px;
  `;

  const BorderRow = styled(Row)`
    border-bottom: 1px solid black;
  `;

  return (
    <BorderRow>
      <StyledCol md={{ size: 3, order: 2, offset: 3 }}>
        <Suggester name="municipality"  description="Select a municipality" />
      </StyledCol>
    </BorderRow>
  );
};

export default FilterRow;