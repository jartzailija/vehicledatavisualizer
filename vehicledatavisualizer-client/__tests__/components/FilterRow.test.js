import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

// Component to be tested
import FilterRow from '../../src/components/FilterRow';

describe('<FilterRow />', () => {
  describe('render()', () => {
    test('renders the component', () => {
      const wrapper = shallow(<FilterRow />);
      const component = wrapper.dive();

      expect(toJson(component)).toMatchSnapshot();
    });
  });
});