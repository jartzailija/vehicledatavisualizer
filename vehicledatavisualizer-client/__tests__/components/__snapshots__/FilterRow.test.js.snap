// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FilterRow /> render() renders the component 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bwzfXH",
        "isStatic": true,
        "rules": Array [
          "
    border-bottom: 1px solid black;
  ",
        ],
      },
      "displayName": "Styled(Row)",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bwzfXH",
      "target": [Function],
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
>
  <Styled(Col)
    md={
      Object {
        "offset": 3,
        "order": 2,
        "size": 3,
      }
    }
  >
    <Connect(Suggester)
      description="Select a municipality"
      name="municipality"
    />
  </Styled(Col)>
</StyledComponent>
`;
